keybase/platform_windows_test.go

```diff
@@ -16,11 +16,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func getWinTestPath() string {
```

Comment:
make this `windowsTestPath` ... not really supposed to use `get` in golang  (this is debated tho)